AzureCalling/Assets.xcassets/Colors/gray900.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
We added gray900 here but I don't think we used it anywhere yet.

Reply:
Is .primary color using FluentUI's color asset? If so, I believe we don't need to include gray900 in the hero app's asset?

Reply:
Sure, I can remove it for now.